ra_aid/chat_models/deepseek_chat.py

```
@@ -10,8 +10,8 @@
 class ChatDeepseekReasoner(ChatOpenAI):
     """ChatDeepseekReasoner with custom overrides for R1/reasoner models."""
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, *args, timeout: int = 180, max_retries: int = 5, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)